x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/RollupResponseTranslator.java

```diff
@@ -77,15 +78,30 @@ public static SearchResponse verifyResponse(MultiSearchResponse.Item normalRespo
      * on the translation conventions
      */
     public static SearchResponse translateResponse(MultiSearchResponse.Item[] rolledMsearch,
-                                                 InternalAggregation.ReduceContext reduceContext) {
+                                                 InternalAggregation.ReduceContext reduceContext) throws Exception {
+
+        assert rolledMsearch.length > 0;
+        List<SearchResponse> responses = new ArrayList<>();
+        for (MultiSearchResponse.Item item : rolledMsearch) {
+            if (item.isFailure()) {
+                Exception e = item.getFailure();
+
+                // We can tolerate missing indices, assuming there is at least one to process at the end
+                if (e instanceof IndexNotFoundException) {
+                    logger.warn("Rollup index not found during rollup search.", e);
```

Comment:
I wonder how it is possible to get an `IndexNotFoundException` since we validate that the indices exist before sending the request ? Not sure that logging a warning would help here, if it's not a normal case we should think of a way to convey this information in the response.